rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -505,7 +505,7 @@ def test_globaldb_pragma_foreign_keys(globaldb):
     fails we know that something has changed and we will need to adjust our strategy.
     """
     cursor = globaldb.conn.cursor()
-    cursor.execute('PRAGMA foreign_keys = OFF;')
```

Comment:
also this test fails in multiple places. You did not bother to actually run the tests of the code you were touching?

Reply:
fixed